lib/spanglish.js

```diff
@@ -0,0 +1,9 @@
+module.exports = function spanglish(sentence) {
+	var sentenceArray = sentence.split(" ");
+	for(var i=0; i < sentenceArray.length; i++) {
+		sentenceArray[i] = sentenceArray[i].replace(/[\.,-\/#!$%\^&\*;:{}=\-_`~()]/g,"");
```

Comment:
Nice way to handle unwanted characters. You can also check out something called a Regex; it would let you clean this up a bit more.